app/controllers/repos_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ReposController < ApplicationController
+  def index
+    github_service = GithubService.new(current_user.token)
+    @repos = github_service.find_repos
```

Comment:
@jk1dd Kind of depends on what you're comfortable with. If you wrote a feature test around the hash implementation, then the feature test shouldn't need to change (one of the great things about feature tests). Best practice would be to test the new PORO directly as well. In class, I showed a way to stub the API call to test the model.  If you're short on time, I'd hold off on that though since your feature test kind of tests it already. Testing at the unit level will be easier to test edge cases and those tests run faster than your feature tests. Ultimately having a blend of both is ideal. But hey, this is Turing. Totally understand that you might need to cut a corner. I'd cut the model layer test in this scenario if you need to.